### DISCUSSION TITLE
Implement OracleConfigurationProviders (file, https)

### DISCUSSION THREAD

**Discussion by jheiselman** [oracle/python-oracledb#365] [Ideas]
Are there currently plans to implement the new OracleConfigurationProvider API in the python Oracle driver like it implemented in the OJDBC driver?

https://docs.oracle.com/en/database/oracle/oracle-database/23/jajdb/oracle/jdbc/spi/OracleConfigurationProvider.html

**Comment by anthony-tuininga** [comment]
It is on the list of things we are considering, yes. Feel free to turn this into an enhancement request!

**Reply by jheiselman** [reply]
Submitted ER #366 as suggested.
